### PR TITLE
Make Offer-page use more text keys

### PR DESCRIPTION
**Pull request by hedvigoscar**

### CODE REVIEW

File: src/pages/Offer/sections/HedvigSwitch.tsx [hunk 1]
```diff
@@ -113,13 +89,33 @@ export const HedvigSwitch: React.SFC = () => (
...
           <Row>
-            {COLUMNS.map((col) => (
-              <Col key={col.key}>
-                <ImageIcon src={col.image} />
-                <Title>{col.title}</Title>
-                <Paragraph>{col.paragraph}</Paragraph>
-              </Col>
-            ))}
+            <Col>
+              <ImageIcon src="assets/offering/hedvig-dot-1.svg" />
```
Comment (palmfjord) [inline src/pages/Offer/sections/HedvigSwitch.tsx]: /assets? :)

Comment (palmfjord) [reply src/pages/Offer/sections/HedvigSwitch.tsx]: (applies to all)

File: src/pages/Offer/sections/InsuredAmount.tsx [hunk 2]
```diff
@@ -80,12 +57,54 @@ export const InsuredAmount: React.SFC = () => (
...
+                <TranslationsConsumer textKey="OFFER_INSURED_AMOUNT_COL_FOUR_TITLE">
+                  {(title) => title}
+                </TranslationsConsumer>
+              </Col>
+              <Col>
+                <TranslationsConsumer textKey="OFFER_INSURED_AMOUNT_COL_FOUR_AMOUNT">
+                  {(amount) => amount}
+                </TranslationsConsumer>
+              </Col>
+            </Row>
```
Comment (frapperino) [inline src/pages/Offer/sections/InsuredAmount.tsx]: Maybe still use a map for a text keys arr for readability in code and less repeating?

Comment (hedvigoscar) [reply src/pages/Offer/sections/InsuredAmount.tsx]: I'm inclined to agree. @palmfjord thoughts?

Comment (palmfjord) [reply src/pages/Offer/sections/InsuredAmount.tsx]: yes, i agree 😬

Comment (frapperino) [reply src/pages/Offer/sections/InsuredAmount.tsx]: nice:) notice the section with placeholder images (Det här är hedvig) has been changed in master to the real ones.

File: src/pages/Offer/sections/HedvigInfo.tsx [hunk 3]
```diff
@@ -80,6 +80,28 @@ const Paragraph = styled('p')({
   color: colors.DARK_GRAY,
 })
 
+const cols: ReadonlyArray<{
```
Comment (palmfjord) [inline src/pages/Offer/sections/HedvigInfo.tsx]: `ReadonlyArray` ❤! We should use this more often

File: src/pages/Offer/sections/HedvigSwitch.tsx [hunk 4]
```diff
@@ -76,6 +76,28 @@ const ImageIcon = styled('img')({
   position: 'relative',
 })
 
+const cols: ReadonlyArray<{
+  imageUrl: string
+  title: string
+  paragraphKey: string
+}> = [
+  {
+    imageUrl: '/assets/offering/hedvig-dot-1.svg',
```
Comment (palmfjord) [inline src/pages/Offer/sections/HedvigSwitch.tsx]: oops

Comment (palmfjord) [reply src/pages/Offer/sections/HedvigSwitch.tsx]: (new-member-assets)

Comment (hedvigoscar) [reply src/pages/Offer/sections/HedvigSwitch.tsx]:
```suggestion
    imageUrl: '/new-member-assets/offering/hedvig-dot-1.svg',
```

**Pull request merged**